frontend/views/components/modal/ModalTemplate.vue

```diff
@@ -14,10 +14,10 @@
       .c-modal-content(ref='card' v-if='modalIsActive')
         header.c-modal-header(
           :class='{ "has-subtitle": $scopedSlots.subtitle }'
-          v-if='$scopedSlots.title || $scopedSlots.subTitle'
+          v-if='$scopedSlots.title || $scopedSlots.issubtitle'
```

Comment:
@pieer this should be `$scopedSlots.subtitle`

Reply:
Of course... I'm tired today...